src/main/java/com/cloudbees/jenkins/plugins/bitbucket/BitbucketSCMSource.java

```diff
@@ -1450,11 +1488,16 @@ public BitbucketProbeFactory(BitbucketApi bitbucket, BitbucketSCMSourceRequest r
             this.request = request;
         }
 
-        @NonNull
+        @Nonnull
```

Comment:
Please don't use `javax.annotation`, License issue and JSR 305 issue.
https://blog.codefx.org/java/jsr-305-java-9/#comment-3582702088